packages/dev-tools-pages/ts/components/Animations/Trace/index.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+
+import { BaseAnimation } from '../index';
+
+import * as animationData from './data.json';
+
+const AnimationTrace: React.StatelessComponent<{}> = () => (
+    <BaseAnimation animationData={animationData} width={2241} height={610} />
+);
+
+export { AnimationTrace as Animation };
```

Comment:
Why are all these Components being named `Animation`?

Reply:
I name all of them `Animation` so that I can do the same logic in the `react-loadable` component https://github.com/bakkenbaeck/0x-monorepo-1/blob/dev-tools-pages/packages/dev-tools-pages/ts/pages/Trace.tsx#L22-L25. Do you want me to change the names of the export?

Reply:
I can't see the link and can't tell how much more work that would be... but yes all other things being more or less equal would love if components had unique names.

Reply:
Can't the `as Animation` casting happen when this component is imported? Seems more appropriate.